src/main/java/org/example/Main.java

```diff
@@ -6,26 +6,24 @@
 public class Main {
 
     public static final String CORRECT_NUMBER = "5";
-    public static  int MAX_TRIES = 5;
-    public static int NUMBER_TRIES= 0;
+    public static final int MAX_TRIES = 5;
+
 
     public static void main(String[] args) {
-        while (NUMBER_TRIES < MAX_TRIES) {
-            Scanner number = new Scanner(in);
-            out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
-            String name = number.nextLine();
-            out.println(name);
-            if (CORRECT_NUMBER.equals(name)) {
-                out.println("Wszedles");
-                break;
-            } else {
-                NUMBER_TRIES++;
-                out.println("Nie prawidlowe haslo");
+        int numberTries= 0;
+        Scanner number = new Scanner(in);
+        while (numberTries < MAX_TRIES) {
+            out.printf("Wpisz " + CORRECT_NUMBER + " lub inny numer:");
```

Comment:
You've replaced the method but didn't use it's functionality.
read how to use formatting